src/modules/hooks/useBlockExplorer.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react';
```

Comment:
Could you move the hook and its tests under a `/hooks/useBlockExplorer` folder?

Reply:
✅ https://github.com/aragon/ods/pull/220/commits/d62edbc6941044ca90b041375055a4a9f610587f